cli/tools/test_runner.rs

```diff
@@ -220,22 +221,6 @@ pub async fn test_specifier(
     test_source.push_str(&format!("import \"{}\";\n", main_module));
   }
 
-  test_source
-    .push_str("await new Promise(resolve => setTimeout(resolve, 0));\n");
```

Comment:
Why is this tick removed and what was its purpose?

Reply:
Removed because tests are passing without it.

This test is the one that used to fail without the tick skip: https://github.com/denoland/deno/blob/main/cli/tests/testdata/test/clear_timeout.ts

Reply:
Thanks that makes sense